okapi-core/src/test/java/org/folio/okapi/MultiTenantTest.java

```diff
@@ -335,6 +335,27 @@ public void test1() {
     ja = new JsonArray(res.body().asString());
     Assert.assertEquals(3, ja.size()); // two sample modules and auth module
 
+    res = given()
+        .header("Content-Type", "application/json")
+        .header("X-Okapi-Tenant", tenant2)
+        .get("/_/proxy/tenants/" + tenant2 + "/modules")
+        .then().statusCode(401).log().ifValidationFails()
+        .extract().response();
+
+    res = given()
+        .header("Content-Type", "application/json")
+        .header("X-Okapi-Token", okapiTokenTenant2)
+        .get("/_/proxy/tenants/" + tenant2 + "/modules")
+        .then().statusCode(200).log().ifValidationFails()
+        .extract().response();
+
+    res = given()
+        .header("Content-Type", "application/json")
+        .header("X-Okapi-Token", okapiTokenTenant2)
+        .get("/_/proxy/tenants/" + tenant1 + "/modules")
+        .then().statusCode(200).log().ifValidationFails()
```

Comment:
Does this mean one tenant can see another tenant's modules?

Reply:
No. In this test, the permissionsRequired are in effect and real mod-authtoken would reject because the tenant does not have permissions.